x-pack/test/saved_object_api_integration/common/lib/space_test_utils.ts

```diff
@@ -4,16 +4,38 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { DEFAULT_SPACE_ID } from '../../../../plugins/spaces/common/constants';
+/* eslint-disable prettier/prettier */
```

Comment:
I was toying with that to prevent prettier from adding a bunch of extra line breaks, since this is just a group of utilities for the integration tests. I'll take this out so prettier is enabled.